theme.toml

```diff
@@ -3,9 +3,9 @@ description = "A zola theme designed for online courses or tutorials"
 license = "GPL-3.0"
 homepage = "https://github.com/elegaanz/zola-theme-course"
 # The minimum version of Zola required
-min_version = "0.15.0"
+min_version = "0.17.1"
 # An optional live demo URL
-demo = "https://c.gelez.xyz"
+demo = "https://zola-themes-demos.github.io/course/"
```

Comment:
I would like to host the demo myself if possible. I migrated it to a new server recently, that's probably why it is broken, I will try to see how to fix it. Could you revert this change in the meantime please?

Reply:
sure no problem.